exception.py

```diff
@@ -0,0 +1,37 @@
+"""
+Define your custom class for exceptions. Show an example how it might be used.
+"""
+
+
+class Error(Exception):
+    pass
+
+
+class OtherTypeError(Error):
+    pass
+
+
+class DivideByZero(Error):
+    pass
+
+
+def zero(a, b):
+    try:
+        if isinstance(a, str) or isinstance(b, str):
+            raise OtherTypeError('Invalid type')
+        elif int(b) == 0:
+            raise DivideByZero('Division by zero')
+        else:
+            print("{} / {} = {}".format(a, b, (a / b)))
+    except OtherTypeError as ddt:
```

Comment:
what is `ddt`?

Reply:
abbreviation

Reply:
of what?

Reply:
OtherTypeError it ote:

Reply:
Do you think it's clear enough for others to understand your abbreviations?

Reply:
If a variable was used several times, then it should be renamed

Reply:
each variable should be named with a correct and clear name. It doesn't matter how many times it is used.

Reply:
done